### DISCUSSION TITLE
Lost License

### DISCUSSION THREAD

**Discussion by MrZeLee** [noah-nuebling/mac-mouse-fix#862] [Questions] [answered]
Hi, I reinstalled the mac mouse fix app in my mac, and it's asking for my license. I already search my email for the license and cannot find it. The license is sent by email? If not, can anybody remind me of what way normally the license is received?

Is there a way to ask for the previous bought license?

thank you in advance

**Comment by noah-nuebling** [comment, marked as answer]
Hi there @MrZeLee, thaks for reaching out. 

Normally, the sales provider Gumroad.com will send an email to you containing the license key. Maybe it ended up in your spam folder or something? Either way, I'll send a new license to your email.

I hope that helps, and let me know if you have any more questions.

**Reply by damingsu** [reply]
I'm sorry I lost my license key too. I've searched all my email, but cannot found it . Maybe I already clear the past emails , so how can I get my license key ?
![image](https://github.com/user-attachments/assets/8ea27b7a-8b54-4842-8811-c6f1b7f515a4)

**Reply by noah-nuebling** [reply]
@damingsu, I'm sorry for the late reply! Please send me an [email](https://redirect.macmousefix.com/?target=mailto-noah), and we'll figure something out.
